lib/lambda-functions/user-interface-handlers/src/permissionSetApi.ts

```diff
@@ -154,6 +154,51 @@ export const handler = async (
           functionLogMode
         );
         permissionSetName = payload.permissionSetData.permissionSetName;
+        /**
+         * AWS SSO enforces that the count of both AWS and customer managed
```

Comment:
Is it a good idea to run these checks? Shouldn't we rely on the AWS API to fail when we attempt an unsupported operation? (what if AWS decides to increase this quota to 20?)

Reply:
I did not consider that the service might increase the quota at a later time. As an account assignment operation would fail when more than 10 policies are being attached to an IAM role i.e. permission set, I put in this check. But, I see your point. Will revert these checks, and push the commit after accommodating any other feedback you have